reference/annotation/acl.rst

```diff
@@ -0,0 +1,96 @@
+@Acl
+====
+
+The ``@Acl`` annotation is used to create a new access control list and to protect the controller
+that is annotation with this ACL accordingly:
+
+.. code-block:: php
+    :linenos:
+
+    /**
+     * @Acl(
+     *     id="user_user_view",
```

Comment:
You forgot to describe an `id` parameter. I think, you can tell that parameter uses as parameter for AclAncestor annotation.